symbols/overlay29.yml

```diff
@@ -3279,6 +3279,17 @@ overlay29:
         
         r0: Entity pointer
         return: True if the monster can see invisible monsters.
+    - name: HasTactic
```

Comment:
I feel like "has" could potentially be interpreted as "is unlocked", which might be confusing. How about something like `IsTacticSet`?

Reply:
Valid, I've renamed it.